frame/fast-unstake/src/mock.rs

```diff
@@ -117,8 +118,9 @@ impl frame_election_provider_support::ElectionProviderBase for MockElection {
 	}
 }
 
-impl frame_election_provider_support::ElectionProvider for MockElection {
-	fn elect() -> Result<frame_election_provider_support::Supports<AccountId>, Self::Error> {
+impl frame_election_provider_support::BoundedElectionProvider for MockElection {
+	type MaxWinners = MaxWinners;
```

Comment:
I think having this type is a mistake as a whole. We should have some fake implementation of `BoundedElectionProvider` and `ElectionProvider` in `election-provider-support` (it probably already exist) and use it in such places. 

Creating a new `MockElection` or similar just to do nothing is probably wrong.

Reply:
You should make `NoElection` be generic over its `MaxWinners` though, in order to make it work here.